borgmatic/config/load.py

```diff
@@ -98,7 +99,15 @@ def __init__(self, preserve_quotes=None, loader=None):
     yaml = ruamel.yaml.YAML(typ='safe')
     yaml.Constructor = Include_constructor_with_include_directory
 
-    return yaml.load(open(filename))
+    with open(filename) as f:
```

Comment:
Nit: No single-letter variables. (I realize this is kind of obnoxious. :smile:)

Reply:
Ah, I keep forgetting 😅. Is there maybe a flake8 rule or something for this? The consistency makes sense to me. And we might add one for using `f"` strings instead of `.format`  if that's consistent too.

Reply:
found this - https://pypi.org/project/flake8-variables-names/ and this - https://github.com/MichaelKim0407/flake8-use-fstring. They don't seem to be that popular though.

Reply:
Yes, I was just looking at them too after your first comment! I'll integrate one or both of these or update https://torsion.org/borgmatic/docs/how-to/develop-on-borgmatic/ page to mention these conventions.

Reply:
That was a lot of f'strings indeed 😆!